src/Microsoft.DocAsCode.Build.Engine/Incrementals/IncrementalBuildContext.cs

```diff
@@ -10,21 +10,23 @@ namespace Microsoft.DocAsCode.Build.Engine.Incrementals
     using Microsoft.DocAsCode.Common;
     using Microsoft.DocAsCode.Plugins;
 
-    public class IncrementalBuildContext
+    internal class IncrementalBuildContext
     {
         private DocumentBuildParameters _parameters;
-
         private Dictionary<string, Dictionary<string, BuildPhase?>> _modelLoadInfo = new Dictionary<string, Dictionary<string, BuildPhase?>>();
+        private Dictionary<string, ChangeKindWithDependency> _changeDict = new Dictionary<string, ChangeKindWithDependency>();
+
+        public string BaseDir { get; private set; }
 
-        public string BaseDir { get; set; }
+        public string LastBaseDir { get; private set; }
 
-        public string LastBaseDir { get; set; }
+        public DateTime? LastBuildStartTime { get; private set; }
```

Comment:
get it!

---
In reply to: [88604259](https://github.com/dotnet/docfx/pull/906#discussion_r88604259) [](ancestors = 88604259)